refactor: Rework the App Record Replay docs into new standard structure (#1243)
edgexfoundry/edgex-docs: docs_src/microservices/application/services/AppRecordReplay/Configuration.md

# App Record and Replay

## Configuration

> This service does not have any addition configuration to that which is provided by the App SDK. 

Reviewer: ```suggestion
This service uses the configuration provided by App SDK with no additional parameters.
```

Author: How about
```suggestion
This service does not have addition configuration to that provided by the App SDK. 
```

Author: Changed to my suggesting.
